.gitignore

```diff
@@ -78,8 +78,8 @@ system-tests/lib/fixtureDirs.ts
 # from npm/webpack-dev-server
 /npm/webpack-dev-server/cypress/videos
 
-# from runner-ct
-/packages/runner-ct/cypress/screenshots
+# from runner
+/packages/runner/cypress/screenshots
```

Comment:
delete. this is already included on line 40

Reply:
Done.